sandbox.sh

```diff
@@ -194,6 +194,28 @@ start_deku_cluster() {
   done
 }
 
+deploy_dummy_ticket() {
+  contract=$(ligo compile contract ./dummy_ticket.mligo)
+  tezos-client --endpoint $RPC_NODE originate contract "dummy_ticket" \
+    transferring 0 from myWallet \
```

Comment:
This may be a bit late to notice, but: where does the alias `myWallet` come from? I didn't see it in the docs.

Reply:
`myWallet` is created during `./sandbox.sh setup`.

Reply:
There is an implicit dependency graph in the commands - not every order you could run them in works. Right now I think it's best if our team just understands this dependency order (it's very intuitive if you understand what the commands are doing), and runs them manually. We should eventually have a `Makefile` like dependency graph for tasks, but I think we should rewrite `./sandbox.sh` in OCaml before we do that.

Reply:
I say this because if you run `./sandbox.sh deploy-dummy-ticket` before running `./sandbox.sh setup` you will get an error.